### ISSUE TITLE
Installation Guidance:

### ISSUE THREAD

**Issue by idvorkin** [vladimir-kotikov/clink-completions#48]
Is there guidance on how you install these completions? 

For examples, right now I'm copying from this git directory into %localappdata%. Is there a better way?

**Comment by CollinChaffin** [comment]
After 2+ yrs I'm a new user and pretty shocked to see this issue, and not a single line in even the readme indicating how on earth one goes about actually installing this.  I see it referenced on a (also years old) pull request to attempt to add a single line (really?) in the readme and a whole page arguing and finally apparently rejecting on how to properly use git to clone a repo.

So, how is this installed?  To what path? What gets read/loaded?  TIA!

**Comment by vladimir-kotikov** [comment]
@CollinChaffin In my own defence I can say that users are mostly supposed to get these completions installed with [Cmder](https://github.com/cmderdev/cmder) (IIRC the latest version of Cmder already includes latest version of these completions) so manual install is not required usually. 

If you need to integrate this into clean clink installation I think you can try putting these files (by cloning repo or downloading archive) to the directory where Clink stores its *.lua files (you can try searching around directories described [here](https://github.com/mridgers/clink/blob/298113a61272afa3f646a912c97e2ce34da67597/docs/clink.md#file-locations)). Also this is an open-source repo so if you find an answer, feel free to comment on existing PR or submit a new one - your efforts would be appreciated! :)

Regards, Vladimir

**Comment by nstrelow** [comment]
https://trzeci.eu/my-setup-for-conemu-and-clink/
Explanation of setting up clink with these scripts here

**Comment by wswind** [comment]
If you installed Clink using the .exe installer. Put the lua files into the folder:
```
%localappdata%\clink
```
If you are using the .zip distribution of Clink . Put the  lua files into a directory called profile  in the same directory where Clink's core files are found.

You can find this in the clink helper.
![image](https://user-images.githubusercontent.com/18399096/74606780-1f9d0e80-510e-11ea-8b00-d0fa52f0e5b1.png)
![image](https://user-images.githubusercontent.com/18399096/74606785-2af03a00-510e-11ea-8b28-ae674cc00c19.png)

**Comment by fallaciousreasoning** [comment]
Using a new version of clink (I'm running v1.1.48) you can run `clink installscripts patch/to/clink-completions`

**Comment by giggio** [comment]
I added this repo as a submodule to my clink installation, and I use a powershell script called in `clink_start.cmd` to register its path if needed.
This is the script:
https://github.com/giggio/clinkfiles/blob/main/install-scripts.ps1
This is where it is called:
https://github.com/giggio/clinkfiles/blob/d6b7f9ff907017d61016512362a76ac80c8ccd90/clink_start.cmd#L5

I hope it helps someone.

**Comment by byteknight** [comment]
> 

**TL;DR:** No need to defend yourself. Help would be nice, but it's all good! Been in your situation before, so I understand! 😉

**"Ad Nauseam" Version:**
Personally I don't think you need to defend yourself if you were kind enough to supply the completions in the first place—_"beggars can't be choosers"_ and all that. Coming from a GNU/Linux environment and being a software engineer for a living, I am quite used to having to figure things out myself when it comes to building, configuring and using source code and software. I'm very grateful for your hard work!

But having said that, it _is_ always nice when code/software is released with a smidgen of "getting started" guidance. Of course, as you said, these were meant for use with Cmder, so I can understand why you felt it wasn't necessary. 😁

In **my** defense (for seeking guidance), however, I am actually using these beautifully crafted completions with ConEmu in a custom setup, as I prefer that over Cmder when I use Windows. The latter is fine, of course, but I prefer the greater level of customization and utility provided by the former, personally.

In short, thanks for the completions, and don't worry about defending yourself. I'm just grateful for the release and I can figure it out or ask elsewhere. 👌

**Comment by chrisant996** [comment]
Added installation directions in 47e75bbc3960a12ab10874cbe33efa2a1c00b9cd.
